frontend/src/app/application-forms/application-noncommercial-group/application-noncommercial-group.component.html

```diff
@@ -90,7 +90,7 @@ <h3 class="usa-alert-heading">There are errors in your application.</h3>
 
         <label class="usa-input" for="name">Event Name</label>
         <input id="name" formControlName="eventName" type="text" aria-required="true" />
-        <app-error-message id="name-error" name="Event name" [control]="applicationForm.controls.eventName"></app-error-message>
+        <app-error-message fieldId="name-error" name="Event name" [control]="applicationForm.controls.eventName"></app-error-message>
```

Comment:
I'm not sure I understand what fiedlID attribute error does?

Reply:
`fieldId` is an input value that passes the ID value to the error message component. I had to change it from `id` to prevent duplicate IDs.